static/app/components/assigneeSelectorDropdown.tsx

```diff
@@ -377,7 +377,7 @@ export class AssigneeSelectorDropdown extends Component<
 
     return (
       <InviteMemberLink
-        to=""
+        to="#invite-member"
```

Comment:
This should just not have the to property no? It’s not a link

Reply:
@evanpurkhiser  I argue that it's a link.

I needed to add a non-empty string for the `to` prop because  `<Link />` in React Router v3 will not add the `href` attribute for empty strings (i.e. `to=""`).

We have tests in RTL such as `screen.findByRole('link', {name: 'Invite Member'})` that will be unable to find `<a />` elements that are missing the `href` attribute. 

Since I've now properly propagated the react router context in tests, we're now using the `<Link />` component in React Router v3, which should resemble how it would be rendered in the actual app.

Previously, the react router context wasn't properly propagated, we've fallen back to the regular `<a />` tag in our tests here with the `href` attribute present: https://github.com/getsentry/sentry/blob/6f1f9fd37d291f74537ce97b043f72f832381bcc/static/app/components/links/link.tsx#L63-L67